### ISSUE TITLE
Javascript errors on full screen image load

### ISSUE THREAD

**Issue by ivernus** [jellyfin/jellyfin#1912]
**Full page images are not displaying**

On ``firefox-69.0.1-3.fc30.x86_64`` and ``google-chrome-stable-77.0.3865.120-1.x86_64`` full page images are not display. Only the controls are shown. 

**To Reproduce**

1. Go to any photos page in Jellyfin
2. Click on thumbnail preview
3. Large screen image controls open but no image appears

**Expected behavior**

A full page image would be shown

**Logs**

These logs are taken from the web developer tools in Firefox. 

```javascript
ReferenceError: Swiper is not definedslideshow.js:212:17
TypeError: item is undefined2 slideshow.js:51:25
    getImgUrl http://X.X.X.X8096/web/components/slideshow/slideshow.js?v=10.4.0:51
    showNextImage http://X.X.X.X:8096/web/components/slideshow/slideshow.js?v=10.4.0:564
    nextImage http://X.X.X.X:8096/web/components/slideshow/slideshow.js?v=10.4.0:296
```

**System (please complete the following information):**
 - OS: Fedora server 30
 - Browser: Firefox and Chrome
 - Jellyfin Version: 10.4.0.1
 - Reverse proxy: no

**Additional context**

No errors appear in the jellyfin logs

I attempted to add the libraries from a CDN to the code of index.html as per https://github.com/JayChase/angular2-useful-swiper/issues/31 but this did not resolve the issue.

**Comment by dkanada** [comment]
jellyfin/jellyfin-web#513

**Comment by ivernus** [comment]
Fixed in jellyfin/jellyfin-web#513
